shopfloor_reception_mobile/static/src/scenario/reception.js

```diff
@@ -0,0 +1,723 @@
+                group_title_default: "Packs available",
+                group_color: this.utils.colors.color_for("screen_step_todo"),
+                list_item_component: "list-item",
+                list_item_options: {
+                    fields: [
+                        {path: "weight", label: "Weight", display_no_value: true},
+                        {
+                            path: "move_line_count",
+                            label: "Lines already in pack",
+                            display_no_value: true,
+                        },
+                        {path: "storage_type.name", label: "Package type"},
+                    ],
+                },
+            };
+        },
+        picking_summary_records_grouped: function (picking) {
+            const grouped_lines = this.utils.wms.group_lines_by_product(
+                picking.move_lines,
+                {
+                    prepare_records: _.partialRight(
+                        this.utils.wms.group_by_pack,
+                        "product"
+                    ),
+                    group_color_maker: function (value) {
+                        return value[0].progress == 100
+                            ? "screen_step_done"
+                            : "screen_step_todo";
+                    },
+                    group_no_title: true,
+                }
+            );
+            // There are two color options for each group: done or todo.
+            // We make sure to display the todo groups on top.
+            return grouped_lines.sort((prev, next) =>
+                prev.group_color > next.group_color ? -1 : 1
+            );
+        },
+        on_search: function (input) {
+            this.filtered_pickings = this.state.data.pickings.filter((picking) =>
+                this._apply_search_filter(picking, input.text)
+            );
+        },
+        on_select_document: function (selected) {
+            this.$root.trigger("scan", selected.name);
+        },
+        on_mark_done: function () {
+            this.$root.trigger("mark_as_done");
+        },
+        lot_has_expiry_date: function () {
+            // If there's a expiry date, it means there's a lot too.
+            const expiry_date = _.result(
+                this.line_being_handled,
+                "lot.expiration_date",
+                ""
+            );
+            return !_.isEmpty(expiry_date);
+        },
+        get_next_line_id_to_handle: function () {
+            // The enpoints in the backend accept multiple selected line ids.
+            // However, in this particular scenario, we only want to deal with one line at a time.
+            // For that, we select the first line we find that hasn't been completely dealt with yet.
+            const next_unhandled_line = this.state.data.selected_move_lines.find(
+                (line) => line.qty_done < line.quantity
+            );
+            return this._get_selected_line_ids([next_unhandled_line]);
```

Comment:
`_get_selected_line_ids` is used only here and only w/ one record, why do you need a list and loop on it to get the id?